lane.py

```diff
@@ -300,7 +301,7 @@ def order_by(self):
         ]
     
         primary_order_by = self.order_facet_to_database_field(self.order)
-        if primary_order_by:
+        if isinstance(primary_order_by, Function) or primary_order_by:
```

Comment:
Could you do this as `primary_order_by is not None`?

Reply:
That doesn't work when `primary_order_by` is a column - the normal case.

Reply:
Although, I tried `!=` rather than `is not`, I don't know if that would make a difference. I think `!=` tries to make it into a clause, which can't be used as a boolean.

Reply:
Yes, `is not` seems to work.